### DISCUSSION TITLE
Less noticeable texture repeat ?

### DISCUSSION THREAD

**Discussion by ThomasCartier** [bevyengine/bevy#14399] [Q&A]
Hi,
Do we have a way to deal with kind of randomization for repeating texture ?
Please have a look [here](https://www.youtube.com/embed/tKK0Pzw3sjM) for what I mean.

Thanks!

**Comment by torsteingrindvik** [comment]
We don't have that, here's an issue for it: https://github.com/bevyengine/bevy/issues/13682
